Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM folioci/alpine-jre-openjdk11:latest
+
+ENV VERTICLE_FILE edge-search-srs-1.0.0-SNAPSHOT.jar
+
```

Comment:
Why is the version number hard-coded? That seems prone to future failure.

Reply:
I used it from other spring based projects.

Reply:
I agree this is bad practice.